spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,36 +32,36 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.0-RC1]]
-=== MicroProfile 4.0 Release Candidate 1 (Q42020)
-// MicroProfile 4.0-RC1 is the 14th platform release for the
+[[microprofile4.0]]
+=== MicroProfile 4.0 (Q12021)
+// MicroProfile 4.0 is the 14th platform release for the
 // https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project].
-MicroProfile 4.0-RC1 is based on Jakarta EE 8, the first MicroProfile release to
+MicroProfile 4.0 is based on Jakarta EE 8, the first MicroProfile release to
 be based on Jakarta EE (replacing the role of Java EE).
 Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with
 Jakarta EE dependencies for all MicroProfile specifications.
 
 The following specifications include API incompatible changes:
 
-* https://download.eclipse.org/microprofile/microprofile-config-2.0-RC1/microprofile-config-spec.html#_incompatible_changes[Config]
-* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-3.0-RC2/microprofile-fault-tolerance-spec.html#_backward_incompatible_changes[Fault Tolerance]
-* https://download.eclipse.org/microprofile/microprofile-health-3.0-RC5/microprofile-health-spec.html#_incompatible_changes[Health]
-* https://download.eclipse.org/microprofile/microprofile-metrics-3.0-RC2/microprofile-metrics-spec-3.0-RC2.html#_breaking_changes[Metrics]
-* https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/microprofile-openapi-spec.html#_incompatible_changes[Open API]
+* https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec.html#_incompatible_changes[Config]
```

Comment:
This highlight another missing task to be done. At the moment, the artifacts are under staging

https://download.eclipse.org/microprofile/staging/microprofile-config-2.0/microprofile-config-spec-2.0.html#_incompatible_changes

There will be another process to move the staging to normal place.
https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec.html#_incompatible_changes

I thought @starksm64 has done some investigation. @starksm64 any updates?

Reply:
@Emily-Jiang is this blocking the merge of this PR?

Reply:
I did a little digging and it seems that this was done on purpose - https://github.com/eclipse/microprofile-parent/pull/31/files. The download directories are created by ssh-ing into eclipse.org so if we want to move the artifacts from `microprofile/staging/microprofile-spec-version` to `microprofile/microprofile-spec-version` (one dir up) then we will need to do this again manually with ssh I guess and thus we would need to do it in Jenkins again because of the correct credentials. Would it make sense to create another Jenkins job "Post release" that would do this?

Reply:
@xstefank we have only done the 1st part to push the artifacts to the staging repo. After the spec is released, they should push the artifacts from staging to the parent dir. This was the jenkins steps to be done. @starksm64 said he would investigate, hence my previous comments. 

> Would it make sense to create another Jenkins job "Post release" that would do this? Of course we can just revert the above PR and put it directly into the right place.
yes, one more step needs to be added to release groovy to move the artifacts from staging to the parent dir.

Reply:
This comment has been resolved under issue #199 by @xstefank !